server/index.js

```diff
@@ -41,17 +43,33 @@ app.post('/reservation', function (req, res) {
   var booking = req.body;
 
   Reservation.make(booking)
-  .then((notification) => {
-    res.write(JSON.stringify(notification));
-    res.end();
-  })
-  .catch((err) => {
-    console.log('Error occurred: ', err);
-    res.status(500).send(new Error(err));
-    res.end();
-  });
+    .then((notification) => {
+      res.write(JSON.stringify(notification));
+      res.end();
+    })
+    .catch((err) => {
+      console.log('Error occurred: ', err);
+      res.status(500).send(new Error(err));
+      res.end();
+    });
 });
 
+//!! RON'S ADDITION FOR THE PUT REQUEST TO UPDATE THE RESERVATION
+//PUT is update: mongo _id field is included in the object res, use
+//Maybe use:
+//https://docs.mongodb.com/manual/reference/method/db.collection.replaceOne/
+//or https://docs.mongodb.com/manual/reference/method/db.collection.update/
+//db.collection.replaceOne(), or db.collection.update()
+app.put('/reservation', function (req, res) {
```

Comment:
if you incorporate a reservation_Id category into your schema, then you can query based off the reservation_Id being passed in and probably would set a route to something like 'reservation/:reservationId'

you could then incorporate delete in the same way